.buildkite/pipeline.yml

```diff
@@ -61,6 +61,8 @@ steps:
           run: http-response-tests
       - docker-compose#v3.7.0:
           run: payload-helper-tests
+      # - docker-compose#v3.7.0:
```

Comment:
A comment to explain why it's commented out would be good, perhaps referencing an internal ticket.